ephios/settings.py

```diff
@@ -14,12 +15,14 @@
 except ImportError:
     import importlib.metadata as importlib_metadata
 
-
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 env = environ.Env()
 # for syntax see https://django-environ.readthedocs.io/en/latest/
-environ.Env.read_env(env_file=os.path.join(BASE_DIR, ".env"))
+# read env file from ENV_PATH or fall back to a .evn file in the project root
+env_path = env.str("ENV_PATH", default=os.path.join(BASE_DIR, ".env"))
+print(f"Loading ephios environment from {Path(env_path).absolute()}")
```

Comment:
sure you want to print stuff instead of logging it?

Reply:
I will try again but it wasn't showing up using logging. Maybe that's because to configure logging, this file needs to be loaded first?

Reply:
![Screenshot_2023-06-27-11-03-31-15_ffb2f5e1b976ff98cfc94f359fbce8de.jpg](https://github.com/ephios-dev/ephios/assets/5970076/38b42597-9040-4ad3-969e-46c81562ac14)

Ja well